test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -267,6 +267,9 @@ json_spirit::mObject fillBCTest(json_spirit::mObject const& _input)
 	if (_input.count("noBlockChainHistory") > 0)
```

Comment:
this field is not needed in filled test. this is an instruction for test filling